localization/ndt_scan_matcher/src/util_func.cpp

```diff
@@ -254,3 +247,28 @@ std::vector<geometry_msgs::msg::Pose> createRandomPoseArray(
 
   return poses;
 }
+
+Eigen::Matrix4f fromRosPoseToEigenMatrix4f(const geometry_msgs::msg::Pose & ros_pose)
```

Comment:
I think `rosPoseToEigenMatrix4f` can make sense

Reply:
Or just `poseToMatrix4f` is also sufficient because we can infer from argument/return types